lnd_test.go

```diff
@@ -7198,6 +7198,85 @@ func testDataLossProtection(net *lntest.NetworkHarness, t *harnessTest) {
 
 	assertNodeNumChannels(t, ctxb, dave, 0)
 	assertNodeNumChannels(t, ctxb, carol, 0)
+
```

Comment:
Least blocking route would be a comment that delineates this new testing scenario.

Reply:
Improved the comments.

Agreed that some of the tests would benefit from being split up. I think this might cause our integration tests to run even longer though, bc of the added overhead. Might be worth looking into in combination with running integration tests in parallel (perhaps using containers?).

Reply:
👍 

Yeah we'll need to white board out a better architecture as we add more backends, and also eventually starting other platforms other than mac OS.